android/build.gradle.kts

```diff
@@ -27,6 +27,7 @@ buildscript {
 plugins {
   id("org.jetbrains.dokka") version "1.8.20"
   id("org.owasp.dependencycheck") version "8.2.1"
+    alias(libs.plugins.org.jetbrains.kotlin.android) apply false
```

Comment:
Do we have a reason for applying this here?

Reply:
this might have been appended here by mistake let me remove it